jwst/jump/jump.py

```diff
@@ -43,7 +53,11 @@ def run_detect_jumps(input_model, gain_model, readnoise_model,
                                     four_grp_thresh, max_cores,
                                     max_jump_to_flag_neighbors,
                                     min_jump_to_flag_neighbors,
-                                    flag_4_neighbors, dqflags.pixel)
+                                    flag_4_neighbors, dqflags.pixel,
+                                    after_jump_flag_dn1=after_jump_flag_dn1,
```

Comment:
Given that all of the function arguments are specified in this call, in the proper order, I don't think it's really necessary to explicitly assign the values to each argument name for the last 4 (new) arguments. Could simply it to just listing the variables that are being passed, i.e. (..., dqflags.pixel, after_jump_flag_dn1, after_jump_flag_n1, ...)

Reply:
Good point.  Fixed.